packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -168,21 +167,31 @@ export class MonthPicker extends React.PureComponent {
 
   render() {
     const { selectAllText, clearAllText } = this.props;
+    const Heading = this.props.headingType
+      ? `h${this.props.headingType}`
+      : `h4`;
     const classes = classNames(
-      'ds-c-month-picker',
       'ds-c-fieldset',
       'ds-u-margin-y--3',
       this.props.className
     );
     return (
-      <div className={classes}>
-        <div className="ds-c-label">
-          <h4 className="ds-u-margin--0" id={this.labelId}>
+      <div className="ds-c-month-picker ds-u-margin-y--3">
```

Comment:
`this.props.className` would need added here too, to align with its intended behavior:

> Additional classes to be added to the root element.

Another thing to note is that this root element and the `fieldset` both have a `y` margin, but only the root element has the option for overriding that margin. So, I think we'd want the margin only applied to the root element so the user can customize it if they'd want.